src/main/java/com/epam/training/auction/integration/CamelConfigurations.java

```diff
@@ -30,8 +30,7 @@ public void configure() throws Exception {
                 from("activemq:queue:auctions").choice()
                         .when(body().isInstanceOf(AuctionTransferObject.class)).bean(auctionsService, "addAuction")
                         .when(body().isInstanceOf(Long.class)).bean(auctionsService, "getAuctionById")
-                        .when(method(auctionsService, "getActiveAuctions")).bean(auctionsService, "getActiveAuctions")
-                        .when(method(auctionsService, "getArchivedAuctions")).bean(auctionsService, "getArchivedAuctions")
+                        .when(method(auctionsService, "getAuctionsWithActive")).bean(auctionsService, "getAuctionsWithActive")
```

Comment:
here should be something like 
                         .when(body().isInstanceOf(Boolean.class)).bean(auctionsService, "getAuctionsWithActive")